.cloudbees/workflows/workflow.yml

```diff
@@ -8,15 +8,21 @@ on:
       - "*"
 
 jobs:
-  
+
   build:
     steps:
-      - uses: .
+      - name: Git checkout
+        uses: docker://alpine/git:2.43.0
+        run: |
+          set -x
+          git config --global --add safe.directory `pwd`
+          git clone ${{ cloudbees.scm.repositoryUrl }} .
+          git checkout ${{ cloudbees.scm.sha }}
       - id: test
-        name: Build and test
+        name: Run unit tests
         uses: docker://golang:1.21
         run: |
-          go test ./...
+          go test --cover ./...
       - id: dockerconfig
         name: Configure container registry credentials
         uses: cloudbees-io/configure-oci-credentials@v0
```

Comment:
Can we move this action version to v1?

Reply:
Good point! I pushed that change with another commit. Please re-review!